node/src/Cardano/Node/Manager.hs

```diff
@@ -58,13 +58,13 @@ mkHttpsManagerSettings serverId caChain credentials =
         , clientDebug                   = def
         }
     clientShared = Shared
-        { sharedCredentials     = Credentials [credentials]
+        { sharedCredentials     = Credentials $ maybeToList credentials
         , sharedCAStore         = makeCertificateStore caChain
         , sharedSessionManager  = noSessionManager
         , sharedValidationCache = def
         }
     clientHooks = def
-        { onCertificateRequest = const . return . Just $ credentials
+        { onCertificateRequest = const . return $ credentials
```

Comment:
Hmmm... This actually defy the purpose of using the https manager client. If there's no certificate to verify, how do you intent to use TLS :thinking: ?

Reply:
Ahh… I guess we could just use `mkHttpManagerSettings` 🤦‍♂️ 😄 

Note-to-self: need to have the increased `managerResponseTimeout` on both also